daemon/api.go

```diff
@@ -296,6 +296,7 @@ func UserFromRequest(st *state.State, req *http.Request) (*auth.UserState, error
 type metarepo interface {
 	Snap(string, string, store.Authenticator) (*snap.Info, error)
 	FindSnaps(string, string, store.Authenticator) ([]*snap.Info, error)
+	Updates([]string, store.Authenticator) ([]*snap.Info, error)
```

Comment:
Refreshing is updating. Let's please not draw a distinction between the two as that will be needless terminology for everybody to learn and keep in mind. If we want to draw a distinction, let's say "refresh --no-network" or something semantically equivalent.